edge/pkg/metamanager/metaserver/kubernetes/scope/scope.go

```diff
@@ -46,26 +48,32 @@ func NewRequestScope() *handlers.RequestScope {
 
 		Serializer:     serializer.NewNegotiatedSerializer(),
 		ParameterCodec: scheme.ParameterCodec,
-		//Creater:         nil,
-		Convertor: fakers.NewFakeObjectConvertor(),
-		Defaulter: fakers.NewFakeObjectDefaulter(),
-		Typer:     crdserverscheme.NewUnstructuredObjectTyper(),
-		//UnsafeConvertor: nil,
-		Authorizer: fakers.NewAlwaysAllowAuthorizer(),
+
+		StandardSerializers: make([]runtime.SerializerInfo, 0),
```

Comment:
You could add some comments to clarify why newly member variable entities is used.

Reply:
In the definition of `type RequestScope struct` in `vendor/k8s.io/apiserver/pkg/endpoints/handlers/rest.go`, there are a few variables not specified in `edge/pkg/metamanager/metaserver/kubernetes/scope/scope.go`. There is a risk that those `nil` variables lead to panic issues when they are called in somewhere else. So I initialize them here as well.